Can't comiple

Hi! I'm having trouble compiling BOP on Ubuntu Linux with the Java 8 JDK.

Here's the log: http://paste.ubuntu.com/11062365/

I don't know how to interpret the stacktrace; it says this at one point:

`/home/aj/Desktop/BOP/BiomesOPlenty/build/sources/java/biomesoplenty/common/items/ItemJarFilled.java:129: error: ')' expected
						entityPlayer.setCurrentItemOrArmor(0, new ItemStack(BOPCItems.jarEmpty, 1, 0);`

Is this an issue with BOP or with my setup? Thanks for your help!

This is still happening when I run ./gradlew build:

'warning: [options] bootstrap class path not set in conjunction with -source 1.6
/home/aj/Desktop/BiomesOPlenty/build/sources/java/biomesoplenty/common/items/ItemJarFilled.java:129: error: ')' expected
						entityPlayer.setCurrentItemOrArmor(0, new ItemStack(BOPCItems.jarEmpty, 1, 0);'

Alright, I applied the fix myself, but now when I try to compile I get this error (the old one is gone):

`/home/aj/Desktop/BiomesOPlenty/build/sources/java/biomesoplenty/common/blocks/BlockBOPColorizedSapling.java:75: error: no suitable method found for canBlockStay(World,int,int,int,int)
        if (!this.canBlockStay(world, x, y, z, meta))`

--- a/src/main/java/biomesoplenty/common/blocks/BlockBOPColorizedSapling.java
+++ b/src/main/java/biomesoplenty/common/blocks/BlockBOPColorizedSapling.java
@@ -72,7 +72,7 @@ public class BlockBOPColorizedSapling extends BlockSapling
 	protected void checkAndDropBlock(World world, int x, int y, int z)
     {
     	int meta = world.getBlockMetadata(x, y, z);
-        if (!this.canBlockStay(world, x, y, z, meta))
+        if (!this.isValidPosition(world, x, y, z, meta))
         {
             this.dropBlockAsItem(world, x, y, z, meta, 0);
             world.setBlockToAir(x, y, z);

--- a/src/main/java/biomesoplenty/common/items/ItemJarFilled.java
+++ b/src/main/java/biomesoplenty/common/items/ItemJarFilled.java
@@ -126,7 +126,7 @@ public class ItemJarFilled extends Item
 					}
 					else
 					{
-						entityPlayer.setCurrentItemOrArmor(0, new ItemStack(BOPCItems.jarEmpty, 1, 0);
+						entityPlayer.setCurrentItemOrArmor(0, new ItemStack(BOPCItems.jarEmpty, 1, 0));
 					}
 				}
 

